lib/mix/test/mix/tasks/deps_test.exs

```diff
@@ -251,6 +251,12 @@ defmodule Mix.Tasks.DepsTest do
     end
   end
 
+  test "fails with message on missing dependencies" do
+    assert_raise Mix.Error, ~r/"mix deps\.unlock" expects dependencies as arguments/, fn ->
```

Comment:
@whatyouhide wow thanks! I just force-pushed once again, this time with `Mix.Project.push DepsApp` in the beginning of the test